package.json

```diff
@@ -82,7 +82,7 @@
     "sinon": "1.17.3",
     "sinon-chai": "2.8.0",
     "style-loader": "0.13.1",
-    "webpack": "1.12.14",
+    "webpack": "1.14.0",
```

Comment:
I have two separate Vagrant boxes - one for the 600-series code, and the other for making hotfix changes and builds, so it doesn't make too big of a problem for me. That said, the change here was non-destructive as far as I can tell, so I wouldn't be opposed to cherry-picking this into `master`.